Vidstreaming/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+      "name": "GURRMAN375D",
+      "id": "446528923535278082"
+    },
+    "url": "vidstreaming.io",
+    "description": {
+      "en": "A Test."
```

Comment:
Can you change this to something more descriptive?

Reply:
Didn't realize that was still there. It all fixed now.